app/processing/siti-agri/transform-header.js

```diff
@@ -0,0 +1,38 @@
+const { sfi, sfiPilot, lumpSums } = require('../../schemes')
+
+const transformHeader = (headerData, schemeId) => {
+  switch (schemeId) {
+    case sfi.schemeId:
+    case sfiPilot.schemeId:
```

Comment:
Current proposal is SFI mirrors SFI Pilot batch structure and content.  If tweaks are needed, this approach makes it simpler to implement.